src/eq_schema/schema/QuestionnaireFlow/index.test.js

```diff
@@ -0,0 +1,68 @@
+const questionnaire_flow = require(".");
```

Comment:
```suggestion
const QuestionnaireFlow = require(".");
```

This should also be camel case, and classes should start with a capital first letter.